example/server.js

```diff
@@ -140,6 +140,22 @@ function init (config, callback) {
           },
           {
             method: 'POST',
+            path: '/updateOrg',
+            handler: function (request, reply) {
+              var id = request.query.id;
```

Comment:
Interesting idea. 

We would have to store the id on the client side whilst they fill in the form. I assume we will still store the id inside the url? or were you thinking to get rid completely?